ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -17,6 +17,23 @@
 ########################
 
 ### Single Zone Units ###
+AHU_HWSC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
```

Comment:
AHUs typically have some ECON control, you have no model for the mixed air or the outside air damper. This looks the same as your DOAS model. Please confirm what general type this should actually be.